base/range.jl

```diff
@@ -468,6 +468,9 @@ value `r[1]`, but alternatively you can supply it as the value of
 with `TwicePrecision` this can be used to implement ranges that are
 free of roundoff error.
 
+The syntax `a:b` or `a:b:c` where either `a`, `b`, or `c` are floating-points
+creates a `StepRangeLen`.
```

Comment:
Not always: `today():1.0:today()+Year(1)`, though this is true whenever all of `a`, `b`, and `c` are `Real`.

The methods that do this are 
https://github.com/JuliaLang/julia/blob/fde48aed31e436ced411ebf7281a295f27d5b1c7/base/range.jl#L13-L18

Reply:
So, like this isn't applicable or the description should be changed to `when a, b, c are all Real`?

Reply:
It says:

> where either `a`, `b`, or `c` are floating-points

So I think the instance you provided for it not always being true using the `today():1.0:today()+Year(1)` is okay, since `today()` is not a floating-point.